Formula/sile.rb

```diff
@@ -142,13 +138,18 @@ def install
     luapath = libexec/"vendor"
 
     paths = %W[
+      #{Formula["lpeg"].opt_share}/lua/#{luaversion}/?.lua
```

Comment:
Should this have a `?`?

Reply:
Yes, the `?` is a placeholder for arbitrary file basenames.